packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -555,13 +571,18 @@ export default defineComponent({
 
     async deleteLink({ client, share, resource }) {
       this.hideModal()
+      let path = resource.path
+      // sharing a share root from the share jail -> use resource name as path
+      if (this.hasShareJail && path === '/') {
+        path = `/${resource.name}`
+      }
       // removeLink currently fetches all shares from the backend in order to reload the shares indicators
       // TODO: Check if to-removed link is last link share and only reload if it's the last link
       await this.removeLink({
         client,
         share,
-        resource,
-        ...(this.currentStorageId && { storageId: this.currentStorageId })
+        path,
+        storageId: this.highlightedFile.fileId || this.highlightedFile.id
```

Comment:
There are more places in this file where we conditionally add the `storageId: this.currentStorageId`. Can you make it consistent please? What speaks against just returning `this.highlightedFile.fileId || this.highlightedFile.id` from the `currentStorageId` computed prop?